CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Release v0.10.2 (2020-12-23)
+
+- Bump otel dependencies from 0.13.0 to 0.15.1
```

Comment:
This should include otel updates too

Reply:
@MikeGoldsmith the only dependabot update was otel-contrib from 0.15.0 to 0.15.1.

Reply:
Ah, okay - no problem 👍